inc/compat/themes/compat-theme-smart-home.php

```diff
@@ -0,0 +1,159 @@
+<?php
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Compatibility with theme: Smart Home (by Fuel Themes).
+ */
+class FluidCheckout_ThemeCompat_SmartHome extends FluidCheckout {
+
+	/**
+	 * __construct function.
+	 */
+	public function __construct() {
+		$this->hooks();
+	}
+
+
+
+	/**
+	 * Initialize hooks.
+	 */
+	public function hooks() {
+		// Very late hooks
+		add_action( 'wp', array( $this, 'very_late_hooks' ), 100 );
+
+		// Checkout page template
+		add_filter( 'template_include', array( $this, 'checkout_page_template' ), 110 );
```

Comment:
- Why do we need to re-add the checkout page template hooks at higher priority?
- Can we remove the hook from the theme instead? If not, please add a comment to above this line to explain why it is needed.